[BUG] Webkit throws error when packaged within Node Modules using Electron-Packager

**Context:**
- Playwright Version: 1.8.1
- Operating System: Mac
- Node.js version: 14.11.0
- Browser: WebKit
- Extra: Launching a playwright script from Electron

```
  System:
    OS: macOS 10.15.7
    Memory: 79.50 MB / 8.00 GB
  Binaries:
    Node: 14.11.0 - /usr/local/bin/node
    npm: 6.14.8 - /usr/local/bin/npm
  Languages:
    Bash: 3.2.57 - /bin/bash
  npmPackages:
    playwright: ^1.8.1 => 1.8.1 
```


**Describe the bug**
I am working on an Electron app that launches Playwright webkit. The app is being packaged and should work without the user installing playwright themselves. I have packaged the browsers in Node Modules using "PLAYWRIGHT_BROWSERS_PATH=0" and set "process.env.PLAYWRIGHT_BROWSERS_PATH = 0" in my main.js.

Everything works on my local machine but when trying to package the app using Electron-Packager i get the following error:

```

> InstaBot@1.0.0 package-mac /Users/adamlosgood/development/CleanScraper
> electron-packager . --overwrite --platform=darwin --arch=x64  --prune=true --out=release-builds

Packaging app for platform darwin x64 using electron v4.1.4
ENOENT: no such file or directory, stat 'node_modules/playwright/.local-browsers/webkit-1423/WebKitLegacy.framework/WebKitPluginAgent'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! InstaBot@1.0.0 package-mac: `electron-packager . --overwrite --platform=darwin --arch=x64  --prune=true --out=release-builds`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the InstaBot@1.0.0 package-mac script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/adamlosgood/.npm/_logs/2021-02-16T14_31_17_329Z-debug.log
```

I do have .local-browsers/webkit-1423/WebKitLegacy.framework/WebKitPluginAgent in my Node Modules. Everything works fine, and the app can be packaged if the Playwright browsers are installed outside of Node Modules but this will not work for my purpose.

I have already tried deleting and reinstalling my node modules. My issue seems _vaguely_ similar to #2539 and #1988 but none of the offered solutions helped and my problem seemed different enough to warrant a new issue.

Any help would be GREATLY appreciated, as I've been stuck on this for quite a while.

--- a/browser_patches/webkit/BUILD_NUMBER
+++ b/browser_patches/webkit/BUILD_NUMBER
@@ -1,2 +1,2 @@
-1437
-Changed: dgozman@gmail.com Fri Feb 19 16:12:07 PST 2021
+1438
+Changed: lushnikov@chromium.com Fri Feb 19 21:08:07 PST 2021

--- a/browser_patches/webkit/archive.sh
+++ b/browser_patches/webkit/archive.sh
@@ -131,6 +131,9 @@ createZipForMac() {
   # copy protocol
   node $SCRIPTS_DIR/concat_protocol.js > $tmpdir/protocol.json
 
+  # Remove all broken symlinks. @see https://github.com/microsoft/playwright/issues/5472
+  find "${tmpdir}" -type l ! -exec test -e {} \; -print | xargs rm
+
   # zip resulting directory and cleanup TMP.
   ditto -c -k $tmpdir $ZIP_PATH
   rm -rf $tmpdir

